### PR TITLE
Worked on trackjs issue

### PR DESCRIPTION
**Pull request by sandeep-deriv**
Fixed: Added catch statements for api calls and globalobserver error emitter.
Fixed: npm run test Warnings
Fixed: import sequence

### CODE REVIEW

File: src/botPage/view/TradeInfoPanel/TradeTable.js [hunk 1]
```diff
@@ -128,46 +127,52 @@ const TradeTable = ({ account_id, api }) => {
...
-            const { reference } = row;
-            if (reference === trade.reference) {
-                return {
-                    contract_status: translate('Settled'),
-                    contract_settled: true,
-                    reference,
-                    ...trade,
-                };
+        try {
+            const contract_info = await _api.send({ proposal_open_contract: 1, contract_id });
```
Comment (maryiafrantsava-binary) [inline src/botPage/view/TradeInfoPanel/TradeTable.js:132]:
```suggestion
            const contract_info = await _api.send({ proposal_open_contract: 1, contract_id }).catch(e => {
                globalObserver.emit('Error', e);
            });
```

**Review by maryiafrantsava-binary — CHANGES_REQUESTED** [review]

File: src/botPage/view/TradeInfoPanel/TradeTable.js (hunk 1 above)
Comment (maryiafrantsava-binary) [reply src/botPage/view/TradeInfoPanel/TradeTable.js:132]: let me check it in your code and I will be back

File: src/botPage/view/NetworkMonitor.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default class NetworkMonitor {
             this.parentElement.html('<span class=\'connecting\'></span>');
             this.api.send({ ping: '1' }).then(() => {
                 this.parentElement.html('<span class=\'online\'></span>');
-            });
+            }).catch(() => this.parentElement.html('<span class=\'offline\'></span>'));
```
Comment (sara-fs) [inline src/botPage/view/NetworkMonitor.js:22]: Isn't it better to show an error message instead of `offline`?

Comment (sandeep-deriv) [reply src/botPage/view/NetworkMonitor.js:22]: Yes, @sara-fs I think its better to show error, changing it. 👍

**Review by maryiafrantsava-binary — APPROVED** [review]
LGTM👍

**Review by sara-fs — APPROVED** [review]

**Pull request merged**